packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -423,28 +429,23 @@ detail: ${JSON.stringify(eventDetail)}`);
       this.notifyChatMessageEventChange({ message, type: 'chatMessageDeleted' });
 
       // void this._cache.deleteMessage(message.chatId, message);
-      this.notifyStateChange((draft: GraphChatListClient) => {
-        const draftMessage = draft.messages.find(m => m.messageId === message.id) as AcsChatMessage;
-        // TODO: confirm if we should show the deleted content message in all cases or only when the message was deleted by the current user
-        if (draftMessage) this.setDeletedContent(draftMessage);
-      });
     }
   };
 
   /*
    * Event handler to be called when a message edit is received by the notification service
    */
-  private readonly onMessageEdited = async (message: ChatMessage) => {
+  private readonly onMessageEdited = (message: ChatMessage) => {
     if (message.chatId) {
       this.notifyChatMessageEventChange({ message, type: 'chatMessageEdited' });
 
       // await this._cache.cacheMessage(message.chatId, message);
-      const messageConversion = this.convertChatMessage(message);
-      this.updateMessages(messageConversion.currentValue);
-      if (messageConversion.futureValue) {
-        const eventualState = await messageConversion.futureValue;
-        this.updateMessages(eventualState);
-      }
+      // const messageConversion = this.convertChatMessage(message);
```

Comment:
did you want this commented out? or can this be deleted?

Reply:
There are some manipulation code here around messages, I am not sure if would be useful to keep a reference here for now? I can remove it if we think otherwise.

Reply:
If you think it is helpful, leave it and we can remove later.